src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
I think so yeah
On Mar 18, 2016 14:58, "jugarrit" <notifications@github.com> wrote:

> In src/js/smooch.jsx
> <https://github.com/smooch/smooch-js/pull/201#discussion_r56705959>:
>
> > @@ -161,6 +164,14 @@ export class Smooch {
> >                  }
> >              });
> >          }).then((loginResponse) => {
> > +            observable.on('socket:connected', () => {
>
> Would we simply dispatch a conversation action from faye.js?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/smooch/smooch-js/pull/201/files/26e2e78e6c904cddcaa62e3d9f26626112034110#r56705959>
>